extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/stream/YoutubeStreamExtractorAgeRestrictedTest.java

```diff
@@ -19,19 +18,22 @@
 
 import static org.schabi.newpipe.extractor.ServiceList.YouTube;
 
+// We cannot use mocks for this test class because of the content playback nonce field sent in POST
+// bodies requests, which is generated randomly.
```

Comment:
After moving the functions that generate the random strings to the `Utils` class, and having them use a `static` random device, just add a function to the `Utils` class that allows seeding the random device, and then seed it with a hardcoded value in `setUp` functions.

Reply:
Can you guide me more (or better do it)? I am not really good with random numbers.